src/qtgui/dockfft.ui

```diff
@@ -646,6 +646,11 @@
               <property name="currentIndex">
                <number>0</number>
               </property>
+              <item>
+               <property name="text">
+                <string>500 fps</string>
+               </property>
+              </item>
```

Comment:
It would probably make sense to add 333 as well, to fill in the large gap between 250 and 500.